hooli-demo-assets/hooli_demo_assets/assets/sling.py

```diff
@@ -13,6 +14,9 @@ def __init__(self, target_prefix="RAW_DATA"):
         
    def get_group_name(self, stream_definition):
        return "RAW_DATA"
+   
+   def get_tags(self, stream_definition):
```

Comment:
is there no way to infer the storage kind based on the replication config and then grab it in the translator? Not crucial for Hooli but would be a cool feature to show off / know if it's available

Reply:
Good call out—I even hard coded it wrong (S3 instead of Snowflake or DuckDB).

Fixed though.